internal/pkg/initialize/workload_test.go

```diff
@@ -488,7 +488,40 @@ func TestWorkloadInitializer_Service(t *testing.T) {
 				m.EXPECT().AddServiceToApp(&config.Application{
 					Name:      "app",
 					AccountID: "1234",
-				}, "frontend")
+				}, "frontend", true)
+			},
+		},
+		"writes Static Site manifest, and creates repositories successfully": {
```

Comment:
if we are not creating repo for static site - 
```suggestion
		"writes Static Site manifest": {
```